src/src/openthread-core-efr32-config.h

```diff
@@ -32,12 +32,6 @@
  *   efr32 applications for OpenThread.
  */
 
-#include "board_config.h"
-
-// IMPORTANT: Do not remove this include. Apps will build without error, but
-// they will not boot up properly
-#include "em_msc.h"
```

Comment:
- [ ] Double check if this is still needed

Reply:
I have confirmed that removing this no longer breaks functionality via building and testing ot-cli-ftd for a few different boards